frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -333,8 +349,9 @@ local footerTextGeneratorMap = {
     book_chapter = function(footer)
         local chapter_title = footer.ui.toc:getTocTitleByPage(footer.pageno)
         if chapter_title and chapter_title ~= "" then
+            local chapter_title_no_break_spaces = chapter_title:gsub(" ", "\xC2\xA0") -- replace space with no-break-space
```

Comment:
Yes, better, but you can just update this one, no need for a new variable:
```lua
if chapter_title and chapter_title ~= "" then
  chapter_title = chapter_title:gsub(" ", "\xC2\xA0")
```

Reply:
True enough! I wonder if the interpreter is smart enough to optimise that out ... :)